tests/cases/fourslash/server/pasteEdits_noImportNeededInUpdatedProgram.ts

```diff
@@ -0,0 +1,21 @@
+/// <reference path="../fourslash.ts" />
+
+// @Filename: /a.ts
+//// [||]
+
+// @Filename: /b.ts
+//// export const b = 10;
+
+// @Filename: /tsconfig.json
+////{ "files": ["a.ts", "b.ts"] }
+
+const range = test.ranges();
+verify.pasteEdits({
+    args: {
+        pastedText: [
+            `const b = 1;
+console.log(b);`],
+        pasteLocations: [range[0]],

```

Comment:
Two things

1. Make this `test.ranges()` like everywhere else.
2. In the future, if you have an array, use the plural version (`ranges` instead of `range`).